cfgov/cfgov/settings/base.py

```diff
@@ -700,7 +700,10 @@
     'HMDA_OUTAGE': [
         {'condition': 'parameter', 'value': 'hmda-outage', 'required': True},
         {'condition': 'path matches', 'value': r'^/data-research', 'required': True}
-    ]
+    ],
+    
+    # Add HowTo schema markup to answer page
+    'HOW_TO_SCHEMA': []
```

Comment:
Could we add a note like @schbetsy did for the [`HMDA_OUTAGE` flag](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L696-L699) about when the flag can be deleted?

Reply:
It might also be nice to have in that comment the expectation that it's intended to use a `path matches` condition and example.

Reply:
@willbarton Added in 79ca85862359f1f24d33455a95f47bff158b8870

Reply:
Much appreciated!

Reply:
We could provide an example:

```python
# Intended for use with path conditions in the admin, such as:
#    condition: path matches, value: ^/ask-cfpb/
```

Reply:
Example added in 385dc07c962f61193473c77eba9304c21e330993